### PR TITLE
Add org settings API docs

### PR DESCRIPTION
**Pull request by technicallyjosh**
### Proposed changes

- Add org settings API docs
- Fix logo to go back to reference page in docs

### CODE REVIEW

**Review by amineck — DISMISSED** [review]
LGTM!

**Comment by technicallyjosh** [comment]
@usha-mandya @jerae-duffin It looks like our version of redoc doesn't use the `href` property for image linking back to old docs. I can spend some more time figuring that one out if you want or I can put in another PR once figured out 😄

**Review by jcarter3 — APPROVED** [review]

File: docker-hub/api/latest.yaml [hunk 1]
```diff
@@ -1135,6 +1162,60 @@ paths:
...
+        401:
+          $ref: "#/responses/Unauthorized"
+        403:
+          $ref: "#/responses/Forbidden"
+        404:
+          $ref: "#/responses/NotFound"
+    put:
+      summary: Update organization settings
+      description: |
+        Updates an organization's settings. Some settings are only used when the organization is on a business plan.
```
Comment (jerae-duffin) [inline docker-hub/api/latest.yaml:1192]: Updates an organization's settings.  When the organization is on a business plan, it uses specific settings.

**Comment by jerae-duffin** [comment]
Thanks! @technicallyjosh LGTM! Only one suggestion line 1191 and a new PR for the link to docs sounds good to me. =)

File: docker-hub/api/latest.yaml (hunk 1 above)
Comment (technicallyjosh) [reply docker-hub/api/latest.yaml:1192]: Per our discussion, updating to list settings that are on a business plan 😄

**Comment by technicallyjosh** [comment]
Turns out that until we start using Redoc v2, the logo cannot be a link. I'll take a look at upgrading to Redoc v2. Looks like they might support theming now.

**Review by jerae-duffin — APPROVED** [review]
Thanks Josh! LGTM!

**Comment by technicallyjosh** [comment]
@jerae-duffin can you merge it? I don't have the permissions :(

**Pull request merged**